server/routes/prisonerDetailsMiddleware.ts

```diff
@@ -61,7 +63,19 @@ export default function prisonerDetailsMiddleware({ rpService }: Services) {
       }
     }
 
+    try {
+      const config = Config.getInstance()
+      configFile = await config.getConfig()
+      if (!configFile) {
+        logger.warn('No config available, returning false.')
```

Comment:
If we can't get any config I think we need to call `next(err)` to ensure the page errors.